pod.go

```diff
@@ -584,8 +595,8 @@ func (p *Pod) delete() error {
 		return err
 	}
 
-	if state.State != StateReady && state.State != StateStopped {
-		return fmt.Errorf("Pod not ready or stopped, impossible to delete")
+	if state.State != StateReady && state.State != StatePaused && state.State != StateStopped {
```

Comment:
I'm confused - this change is to allow a paused pod to be deleted (killed). Without that line above, you'd have to call `resume()`, then `delete()`...?